src/lib.rs

```diff
@@ -112,6 +117,22 @@ impl Program {
                         return Ok(locals.remove(a as usize));
                     }
                 }
+                Jump(a) => {
+                    iptr += a as isize;
+                    continue;
+                }
+                CondJump(a, b, c) => {
+                    if let Val::B(x) = locals[a as usize] {
```

Comment:
You could probably do a `match` and match `Val::B(true)`, `Val::B(false)`, and `_`, if that looks nicer. Like:
```rust
match locals[a as usize] {
    B(true) => …,
    B(false) => …,
    _ => return …,
}
continue;
```

Reply:
Also you don't need `Val::B` since we've done `use Val::*;`